src/main/java/dk/bec/polonez/reservationsystem/service/AuthService.java

```diff
@@ -0,0 +1,33 @@
+package dk.bec.polonez.reservationsystem.service;
+
+import dk.bec.polonez.reservationsystem.model.User;
+import dk.bec.polonez.reservationsystem.repository.UserRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+@Service
+public class AuthService {
+
+    private final UserRepository userRepository;
+
+    public AuthService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public boolean isAdminLoggedIn() {
+        return getCurrentUser().getRole().getName().equals("SYS_ADMIN");
+    }
+
+    public boolean isPoLoggedIn() {
+        return getCurrentUser().getRole().getName().equals("PLACE_OWNER");
+    }
+
+    public User getCurrentUser() {
```

Comment:
**doubt** Currently all 3 methods in this class are ```public```. I guess you will be calling ```isAdminLoggedIn``` and ```isPoLoggedIn``` from outside of this class so that these 2 methods have to be ```public```. But what about ```getCurrentUser``` - shouldn't it be ```private``` (at least for now)?